views/forms/EditChartForm.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react'
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { withStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+
+import Form from './Form'
+import BarChartFields from './BarChartFields'
+
+import { chartStyles } from '../styles/chart_styles'
+import { getChartForm } from '../fe-utils/fetchUtil'
+
+const ChartForm = ({ classes, handleSubmit, user, graph }) => {
```

Comment:
this should be the same as the chart form we already have. We might need to tweak it slightly to accept initial values or something but we shouldn't need to create a new form as part of this feature.